### PR TITLE
ODS requirement, extending an extension

### PR DESCRIPTION
**Pull request by APageNHS**
ODS requirement is to have a typed period within their org role extension

### CODE REVIEW

File: valuesets/ValueSet-England-PeriodType.xml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+	<contact>
+		<name value="NHS England" />
+		<telecom>
+			<system value="email" />
+			<value value="interoperabilityteam@nhs.net" />
+			<use value="work" />
+			<rank value="1" />
+		</telecom>
+	</contact>
+	<description value="A ValueSet to identify the type of period data being recorded."/>
```
Comment (ryma2fhir) [inline valuesets/ValueSet-England-PeriodType.xml]: Doesnt read right imo. Maybe "A ValueSet to identify the type of data for the date/time period described within the extension."

File: codesystems/CodeSystem-England-PeriodType.xml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+	<caseSensitive value="true"/>
+	<content value="complete"/>
+	<concept>
+		<code value="legal"/>
+		<display value="Legal"/>
+	</concept>
+	<concept>
+		<code value="operational"/>
+		<display value="Operational"/>
+	</concept>
```
Comment (ryma2fhir) [inline codesystems/CodeSystem-England-PeriodType.xml:30]: Would be worth saying where these two codes come from

File: structuredefinitions/Extension-England-OrganisationRole.xml [hunk 3]
```diff
@@ -94,28 +89,26 @@
       <path value="Extension.extension.value[x]" />
       <min value="1" />
       <type>
-        <code value="CodeableConcept" />
+        <code value="Coding" />
```
Comment (ryma2fhir) [inline structuredefinitions/Extension-England-OrganisationRole.xml:92]: Why to code? I thought all our ValueSets where bound to CodeableConcept?

**Review by ryma2fhir — CHANGES_REQUESTED** [review]

File: valuesets/ValueSet-England-PeriodType.xml (hunk 1 above)
Comment (APageNHS) [reply valuesets/ValueSet-England-PeriodType.xml]: Agree, changed

File: codesystems/CodeSystem-England-PeriodType.xml (hunk 2 above)
Comment (APageNHS) [reply codesystems/CodeSystem-England-PeriodType.xml:30]: Agreed, source added.

**Comment by APageNHS** [comment]
ADAM'S TESTING BRANCH, DO NOT APPROVE!!! DO NOT MERGE!!!....

**Comment by APageNHS** [comment]
New branch created for this and naming systems

**Pull request closed without merging**